kubernetes/services/api-gateway.yaml

```diff
@@ -4,16 +4,43 @@ metadata:
   name: api-configmap
 data:
   application.properties: |
+    ignite.thin.client.enable=true
     spring.application.name=alcor-api-gateway
-
     logging.level.org.springframework.web=DEBUG
     logging.level.org.springframework.cloud=DEBUG
     logging.level.com.github.tomakehurst.wiremock=TRACE
     management.endpoints.web.exposure.include=health,prometheus
     spring.sleuth.sampler.probability=1.0
     spring.sleuth.web.skipPattern=(^health.*)
+    ignite.host=ignite-alcor-service.ignite-alcor.svc.cluster.local
+    ignite.port=10800
+    #ignite.key-store-path=keystore.jks
+    #ignite.key-store-password=123456
+    
+    #ignite.trust-store-path=truststore.jks
+    #ignite.trust-store-password=123456
+
     microservices.vpc.service.url=http://vpcmanager-service.default.svc.cluster.local:9001/
     microservices.subnet.service.url=http://subnetmanager-service.default.svc.cluster.local:9002/
+    microservices.port.service.url=http://portmanager-service.default.svc.cluster.local:9006/
+    microservices.sg.service.url=http://sgmanager-service.default.svc.cluster.local:9008/
+    microservices.route.service.url=http://routemanager-service.default.svc.cluster.local:9003/
+    microservices.quota.service.url = http://vpcmanager-service.default.svc.cluster.local:9001/
```

Comment:
This will need to update to a new port after our Quota Mgr check-in (PR #359)

Reply:
NP